src/demo/containers/Demo.tsx

```diff
@@ -126,26 +126,27 @@ class Demo extends React.Component<{}, DemoState> {
   public removeHandler(): void {
     this.setState({ showMarkdown: !this.state.showMarkdown })
   }
-  
+
   public componentDidMount() {
     // Note:  All portals should do this once on the initial app load.
     // This looks for the session token cookie (HttpOnly, unable to directly access), and initialize the session if it does exists.
     SynapseClient.detectSSOCode()
     SynapseClient.getSessionTokenFromCookie()
-    .then((sessionToken: any) => this.handleChange({ token: sessionToken }))
-    .catch((error: any) => {
-      console.error(error)
-    })
+      .then((sessionToken: any) => this.handleChange({ token: sessionToken }))
+      .catch((error: any) => {
+        console.error(error)
+      })
     this.getVersion()
   }
   public render(): JSX.Element {
     let token: string | undefined = ''
     token = ''
     return (
       <div>
-        <p className="App-intro text-center">Synapse production version: {this.state.version}</p>
-        {
-          (this.state.token && this.state.token !== '') &&
+        <p className="App-intro text-center">
+          Synapse production version: {this.state.version}
+        </p>
+        {this.state.token && this.state.token !== '' && (
```

Comment:
is `this.state.token !== ''` needed?  I think `this.state.token` will be falsy if this.state.token == ''